### ISSUE TITLE
Will it work on real time streaming data ?

### ISSUE THREAD

**Issue by ankurdhuriya** [pyannote/pyannote-audio#1208]
I am currently running it on Apple M2 chip it is taking way much time comparing to Colab. Is there a way that pipeline could be modified to streaming data, and combined with some transcription service ?

**Comment by Aliw7979** [comment]
Did you find any solution?

**Comment by ankurdhuriya** [comment]
Yes, you can look at this repo - https://github.com/juanmc2005/StreamingSpeakerDiarization
